scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncher.java

```diff
@@ -362,6 +366,32 @@ private TaskResultImpl getTaskResult(TaskContext context, Stopwatch taskStopwatc
         return taskContextVariableExtractor.getAllVariables(taskContext);
     }
 
+    //    private void addPreviousTaskLogsFromUserSpace(File taskLogFile) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)